src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -3174,15 +3172,29 @@ public void setTicksPerSpawns(@NotNull SpawnCategory spawnCategory, int ticksPer
 	@Override
 	public int getSpawnLimit(@NotNull SpawnCategory spawnCategory)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		Preconditions.checkArgument(spawnCategory != null, "SpawnCategory cannot be null");
+		Preconditions.checkArgument(spawnCategory != SpawnCategory.MISC,
```

Comment:
This check does not consider if a new spawn limit has been implemented in paper. It would be good if something like an UnimplementedOperationException gets thrown whenever that is the case.